impress-org/givewp
# Feature: add option to disable one time donations for recurring enabled forms

## Description



This updates the donation amount recurring donation settings to:
1. Add setting for enable/disable one-time donations.
2. Remove donation choice in favor of implicit derivative
3. Update labels for admin facing terms "Frequency" and "Donations"

## Affects



The amount block and the donation form.

## Visuals



<img width="1084" alt="Screenshot 2023-08-24 at 12 25 43 PM" src="https://github.com/impress-org/givewp/assets/10138447/657e55f4-2829-424b-94f1-85efab67a42e">

## Testing Instructions



- Create a new v3 form w/ recurring installed
- Enable recurring and toggle the new settings
- Publish changes and make sure the donation form reflects the change and works properly

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205354546701666
  - https://app.asana.com/0/0/1205270388712288
